commands/time.js

```diff
@@ -0,0 +1,37 @@
+/* Usage: .time [zone]
+ * Ex: .time Asia/Beijing, .time JST
+ */
+
+import {Command} from './command.js';
+var tz = require('timezone/loaded');
```

Comment:
change this line to `import timezone from 'timezone/loaded';`, and all `tz` references to `timezone` for me?